packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -94,6 +97,28 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 							{ __( 'Mobile' ) }
 						</MenuItem>
 					</MenuGroup>
+					{ !! window.__experimentalEnableZoomedOutPatternsTab && (
+						<MenuGroup>
+							{ __unstableGetEditorMode() !== 'zoom-out' && (
+								<MenuItem
+									onClick={ () =>
+										__unstableSetEditorMode( 'zoom-out' )
+									}
+								>
+									{ __( 'Zoom Out' ) }
+								</MenuItem>
+							) }
+							{ __unstableGetEditorMode() === 'zoom-out' && (
+								<MenuItem
+									onClick={ () =>
+										__unstableSetEditorMode( 'edit' )
+									}
+								>
+									{ __( 'Zoom In' ) }
+								</MenuItem>
+							) }
```

Comment:
This will probably result in a focus loss. It would be better to toggle the same option and have aria states for it, or have Zoom Out and Zoom In options always available.

Reply:
For some reason Zoom Out and Zoom In is confusing to me when I'm using it. I think offering 100% and 50% as options feels better.

Reply:
https://github.com/user-attachments/assets/d1c82f33-b406-4a0a-8b86-71aa3564f002

Reply:
Let's try desktop and 100% the same thing. 

- 100%
- 50%
- Tablet
- Mobile

Reply:
FWIW Zoom out isn't always 50%